less/_defaults/_spacing-config.less

```diff
@@ -7,14 +7,14 @@
 @item-margin: 0.5rem;
 @item-padding: 1rem;
 
-@item-border-radius: 5px;
+@item-border-radius: 0.25rem;
```

Comment:
0.25rem equates to 4px, not 5. Is this intentional?

Reply:
@cahirodoherty-learningpool , you are correct. Good spot, but yes, it was intensional. @guywillis suggested this would be better and I was in agreement.